asm/istio/istio-operator.yaml

```diff
@@ -33,9 +33,11 @@ spec:
   meshConfig:
     defaultConfig:
       proxyMetadata:
-        GCE_METADATA_HOST: "metadata.google.internal"
```

Comment:
This field doesn't exist in the pkg/istio folder. Should we keep this one?

Reply:
feel free to remove it here. I will update it in the profile to use the cluster info from GKE metadata server.